configure.py

```diff
@@ -1536,7 +1538,8 @@ def clang_inline_threshold():
 else:
     build_artifacts = all_artifacts
 
-status = subprocess.call("./SCYLLA-VERSION-GEN")
+date_stamp = f"-d {args.date_stamp}" if args.date_stamp else ""
```

Comment:
Use --date-stamp so it's clear what's going on.

Reply:
Changed